src/routes/home/components/ProfessionDetails.tsx

```diff
@@ -36,6 +36,8 @@ const ProfessionDetails = ({
   const { data: nextExerciseData, refresh: refreshNextExercise } = useLoadNextExercise(discipline)
 
   const moduleAlreadyStarted = progress !== null && progress !== 0
+  const doneModules =
```

Comment:
(optional) I think it would make sense to use this `doneModules` variable for the progress spinner.